ArraysAndLists/Program.cs

```diff
@@ -19,49 +16,99 @@ public static class ArraysAndLists
         // If more than two people like your post, it returns: "[Friend 1], [Friend 2] and [Number of Other People] others liked your post."
         public static string FacebookLikes(params string[] names)
         {
-            // Delete the below line when you are writing your code, don't forget to use your
-            // own return statement.
-            return default;
-        }
+            if (names.Length == 0)
+                return null;
+            if (names.Length == 1)
+                return names[0] + " liked your post.";
+            if (names.Length == 2)
+                return names[0] + " and " + names[1] + " liked your post.";
+            else
+                return names[0] + ", " + names[1] + " and " + (names.Length - 2) + " others liked your post.";
 
+        }
         // 2 - Write a method that accepts any number of integers (minimum of two integers). If an
         // integer is a duplicate, return "Error". If all integers are unique, then order them in
         // ascending order and return the sorted result as a string of integers separated by spaces.
         public static string NumberSorter(params int[] numbers)
         {
-            // Delete the below line when you are writing your code, don't forget to use your
-            // own return statement.
-            return default;
+            var result = string.Empty;
+
+            var distinct = numbers.Distinct().Count();
```

Comment:
The var distinct should be named something different to better reflect that it means the length of the distinct inputs.
var distinctLength would be  a little better.

This also applies for the var input.
It should be var inputLength.